### PR TITLE
Modernize package

### PR DESCRIPTION
**Pull request by samamorgan**
Reverts soynatan/django-easy-audit#279

### CODE REVIEW

File: easyaudit/migrations/0018_auto_20240105_0125.py [hunk 1]
```diff
@@ -0,0 +1,61 @@
...
+            field=models.CharField(db_index=True, default='', max_length=50, verbose_name='Remote IP'),
+        ),
+        migrations.AlterIndexTogether(
+            name='crudevent',
+            index_together=set(),
+        ),
+        migrations.AddIndex(
+            model_name='crudevent',
+            index=models.Index(fields=['object_id', 'content_type'], name='easyaudit_c_object__82020b_idx'),
+        ),
```
Comment (mschoettle) [inline easyaudit/migrations/0018_auto_20240105_0125.py:60]: This might not be necessary once #258 is merged

File: pyproject.toml [hunk 2]
```diff
@@ -0,0 +1,109 @@
+[tool.poetry]
+name = "django-easy-audit"
+version = "1.3.6-a2"
```
Comment (mschoettle) [inline pyproject.toml:3]: Probably not important for now but consistent:

```suggestion
version = "1.3.6"
```

**Review by mschoettle — APPROVED** [review]
LGTM, added a couple minor comments

File: easyaudit/migrations/0018_auto_20240105_0125.py (hunk 1 above)
Comment (samamorgan) [reply easyaudit/migrations/0018_auto_20240105_0125.py:60]: Yeah it won't, but I'll solve the merge conflict once it lands.

**Comment by jheld** [comment]
@samamorgan looking good. But I'm not sure what the issue with `models.py` is, can you check into it?

**Comment by samamorgan** [comment]
> @samamorgan looking good. But I'm not sure what the issue with `models.py` is, can you check into it?

Fixed the issues, including adding a new migration.

**Pull request merged**